public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -895,7 +895,14 @@ export class PrometheusDatasource
       return uniqueLabels.map((value: any) => ({ text: value }));
     } else {
       // Get all tags
-      const result = await this.metadataRequest('/api/v1/labels');
+      const range = this.timeSrv.timeRange();
+      const start = this.getPrometheusTime(range.from, false);
+      const end = this.getPrometheusTime(range.to, true);
+      const params = {
+        start: start.toString(),
+        end: end.toString(),
+      };
```

Comment:
```suggestion
      const params = this.getTimeRangeParams();
```

I think you can just use this